tbtc-ts/test/deposit.test.ts

```diff
@@ -276,7 +302,10 @@ describe("Deposit", () => {
     })
 
     it("should return script with proper structure", async () => {
-      expect(script.length).to.be.equal(184) // 92 bytes
+      // Returned script should ne the same as expectedDepositScript but
```

Comment:
`ne` -> `be`

Reply:
https://github.com/keep-network/tbtc-v2/pull/112/commits/a82c03ab0e972bfee187f15d51a883bb48f127ba